Docs/source/building/juwels.rst

```diff
@@ -0,0 +1,64 @@
+.. _building-juwels:
+
+Juwels (JSC)
+============
+
+The `Juwels supercomputer <https://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JUWELS/JUWELS_node.html>`_ is located at JSC.
+
+See `this page <https://apps.fz-juelich.de/jsc/hps/juwels/quickintro.html>`__ for a quick introduction.
+
+Installation
+------------
+
+We use the following modules and environments on the system.
+
+.. code-block:: bash
+
+   # please set your project account
+   export proj=<yourProject>
+
+   # required dependencies
+   module load GCC
+   module load OpenMPI
+   module load CUDA
+
+   # AMReX specifics
+   export GPUS_PER_SOCKET=2
+   export GPUS_PER_NODE=4
```

Comment:
:scream: wuha, cannot find them even in the docs: https://amrex-codes.github.io/amrex/docs_html/search.html?q=GPUS_PER_SOCKET&check_keywords=yes&area=default#

Looks like a hint for the build system: https://github.com/AMReX-Codes/amrex/search?q=GPUS_PER_NODE&unscoped_q=GPUS_PER_NODE

Gosh, they should really be runtime and not compile-time options: https://github.com/AMReX-Codes/amrex/blob/38572ef1ff1da8fe89482183f66bf5ba6b91813d/Src/Base/AMReX_GpuDevice.cpp#L201-L217

Reported/inquired in AMReX slack, not relevant for this PR...

Reply:
Yeah, this is handled in the `Make.*` files. They raise a warning, so could we keep them in this PR here, and update it after the discussion with AMReX?

Reply:
They are well placed here, in case JEWELS' job scheduler does not automatically map ranks to GPUs.

I am just surprised that this was not documented. Maybe we can add a slightly longer comment, but that's all.
```bash
# JEWELS; job scheduler does not map ranks to GPUs, so we give a hint to AMReX about the node layout
```